FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+      position: {
+        type: "POPOVER",
+        anchor: e.currentTarget,
+      },
+    });
+  });
+</script>
+```
+
+## Internationalization (i18n)
+
+The Feedback UI currently only comes in English. You can supply your own translation keys that will be used insude the UI by passing an object with your own translation keys.
+
+See [default english localization keys](./src/feedback/config/defaultTranslations.tsx) for a reference of what translation keys can be supplied.
+
+### Static language configuration
+
+If you know the language at page load, you can configure your translation keys while initializing the Bucket SDK:
+
+```javascript
+bucket.init("my-tracking-key", {
+  feedback: {
+    ui: {
+      translations: {
+        // Your translation keys
+      },
+    },
+  },
+});
+```
+
+### Runtime language configuration
+
+If you only know the users language after the page has loaded, you can provide translation keys in the two cases where you open the feedback UI:
+
+### Manual feedback collection
+
+```javascript
+bucket.requestFeedback({
+  ... // Other options
+  translations: {
+    // your translation keys
+  }
+})
+```
+
+### Automated feedback collection
+
+When you are collecting feedback through the Bucket automation, you can intercept the default prompt handling and override the defaults.
+
+If you set the prompt question in the Bucket app to be one of your own translation keys, you can even get a translated version of the question you want to ask your customer in the feedback UI.
+
+```javascript
+bucket.init({
+  feedback: {
+    promptHandler: (message, handlers) => {
+      const translatedQuestion =
+        i18nLookup[message.question] ?? message.question;
+      handlers.openFeedbackForm({
+        title: translatedQuestion,
+        translations: {
+          // your static translation keys
+        },
+      });
+    },
+  },
+});
+```
+
+## Custom styling
+
+You can adapt parts of the looks of the Bucket feedback UI by setting CSS custom properties on your CSS `:root`-scope.
+
+Examples of custom styling can be found in our [development example stylesheet](./dev/index.css).
+
+## Using your own UI to collect feedback
+
+You might have very strict design guidelines in your app, and maybe the Bucket feedback UI doesn"t quite work for you.
+
+In this case, you can actually implement your own feedback collection mechanism, which follows your own design guidelines.
+
+This is the data type you need to collect:
+
+```typescript
+{
+  /** Customer satisfaction score */
+  score?: 1 | 2 | 3 | 4 | 5,
+  comment?: string
+}
+```
+
+Either `score` or `comment` must be defined in order to pass validation in the Bucket tracking API.
+
+### Manual feedback collection
+
+Examples of a HTML-form that collects the relevant data can be found in [feedback.html](./example/feedback/feedback.html) and [feedback.jsx](./example/feedback/feedback.jsx).
+
+Once you have the feedback data, pass it along to `bucket.feedback()`:
```

Comment:
I suggest:

Once you have collected the feedback data, pass it along to `bucket.feedback()`:

Reply:
```suggestion
Once you have collected the feedback data, pass it along to `bucket.feedback()`:
```